### PR TITLE
1.4/1151 uncentred logo

### PR DESCRIPTION
**Pull request by Nikomus**
- Some CSS and blade class shuffling
- Alt text decision logic: https://www.w3.org/WAI/tutorials/images/textual/#image-of-text-used-as-an-unlinked-logo

Fixes https://trello.com/c/duFgVSQj/1151-uncentred-logo

arcv-store test_dashboard

### CODE REVIEW

File: resources/views/store/partials/masthead.blade.php [hunk 1]
```diff
@@ -11,18 +11,18 @@
             </form>
         </div>
     @endif
-    <div class="logo">
```
[the remark this answers is not in the document]
Comment (Nikomus) [reply resources/views/store/partials/masthead.blade.php]: I moved it from the containing div to actual image, to simplify `.logo img` selector to just `.logo`

[the remark this answers is not in the document]
Comment (katjam) [reply resources/views/store/partials/masthead.blade.php]: Of course

File: resources/views/store/partials/masthead.blade.php [hunk 2]
```diff
@@ -11,18 +11,18 @@
...
-        <div>
+        <div class="header-section">
             <ul>
                 <li>User: {{ Auth::user()->name }} </li>
                 <li>Centre:
                     @if( Auth::user()->centres->count() == 1 )
                         {{ Auth::user()->centre->name }}
                     @elseif (Auth::user()->centres->count() > 1)
-                        <form name="centreUserForm" method="post" action="{{ route('store.session.put') }}">
+                        <form name="centreUserForm" id="centre-select" method="post" action="{{ route('store.session.put') }}">
```
[the remark this answers is not in the document]
Comment (katjam) [reply resources/views/store/partials/masthead.blade.php]: Is ok. Just seems a bit hack-y unless other forms have ids.

[the remark this answers is not in the document]
Comment (Nikomus) [reply resources/views/store/partials/masthead.blade.php]: @katjam the only other form I can see that overrides default styling is the remove family one which uses an id to do so. It's more as it's the only instance of this on a page than for specificity.

[the remark this answers is not in the document]
Comment (katjam) [reply resources/views/store/partials/masthead.blade.php]: OK. Discussion closed.

**Pull request merged**